### PR TITLE
Fix log command error

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix https://github.com/PaddlePaddle/cloud/issues/145

### CODE REVIEW

File: go/paddlecloud/logs.go [hunk 1]
```diff
@@ -53,7 +53,8 @@ func (p *LogsCommand) Execute(_ context.Context, f *flag.FlagSet, _ ...interface
 
 	respBody, err := utils.GetCall(utils.Config.ActiveConfig.Endpoint+"/api/v1/logs", queryMap)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "call paddle cloud error %v", err)
+		fmt.Fprintf(os.Stderr, "call paddle cloud error %v, %v", err, respBody)
```
Comment (helinwang) [inline go/paddlecloud/logs.go]: Maybe it's easier for viewing to print `string` instead of `[]byte`, I could be wrong...

Comment (typhoonzero) [inline go/paddlecloud/logs.go]: Printing `string` or `[]byte` can both use `%s`. Newline can also contain in a `string` which will affect the log printing

Comment (helinwang) [inline go/paddlecloud/logs.go]: I see, thanks! Did not know about `%s` can be used on `[]byte`.

Comment (typhoonzero) [inline go/paddlecloud/logs.go]: Yes, I did. both `%v` and `%s` works for `[]byte`

**Pull request merged**